README.md

```diff
@@ -40,10 +40,15 @@ pools:
     listen: /path/to/unix/socket
     user: www-data
     pm: static
```

Comment:
are you sure this works? you assign a string to `pm`, but then treat it as a dict